tests/integration/defs/accuracy/test_chatglm3_gsm8k.py

```
@@ -0,0 +1,382 @@
+
+
+def _write_artifact(name: str, payload: dict) -> str:
+    os.makedirs(ARTIFACT_DIR, exist_ok=True)
+    path = os.path.join(ARTIFACT_DIR, name)
+    with open(path, "w") as f:
+        json.dump(payload, f, indent=2, sort_keys=True)
+    print(f"[gsm8k_artifact] wrote {path}")
+    return path
+
+
+def _run_config(n: int, label: str, cfg: RuntimeCfg, artifact_name: str = None) -> dict:
```

Ruff (0.15.20)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)